e2e/lvmtest/lvmtest.go

```diff
@@ -53,32 +53,32 @@ func createLV(num int, thinpoolSize, virtualSize string) error {
 	for i := 1; i <= num; i++ {
 		prefix := fmt.Sprintf("%s%d/%s", brickPrefix, i, lvmPrefix)
 		brickPath := fmt.Sprintf("%s_mnt", prefix)
-		devicePath := fmt.Sprintf("%s_loop", prefix)
+		devicePath := "/dev/gluster_loop" + strconv.Itoa(i)
```

Comment:
`/dev/gluster_loop` can this be defined as variable above?

Reply:
A constant at the top of the package would be better.

Reply:
A constant at the top of the package would be better.